DESCRIPTION

```diff
@@ -18,7 +18,9 @@ Imports:
     utils,
     graphics,
     methods,
-    deeptime (>= 0.2.1)
+    deeptime (>= 0.2.1),
+    ape,
+    paleotree
```

Comment:
Since we don't actually import anything from `paleotree` and only use it for the examples/tests, we can probably just put it in "Suggests"

Reply:
I tried this and all the tests started failing so I put it back. It might be a syntax error with the way the package is loaded in test_that, but I'm struggling to find any clear guidance, do you know of any?

Reply:
Seems to work now?